ansible/tasks/internal/optimizations.yml

```diff
@@ -0,0 +1,15 @@
+- name: ensure services are stopped
+  community.general.snap:
+    name: amazon-ssm-agent
+    state: absent
+
+- name: ensure services are stopped and disabled for first boot
+  systemd:
+    enabled: no
+    name: '{{ item }}'
+    state: stopped
+  with_items:
+    - snapd
+    - postgresql
+    - postgrest
```

Comment:
should this be `pgbouncer` instead?

Reply:
added pgbouncer; this list is a superset of the things that might be enabled in the image

Reply:
(pgbouncer isn't enabled)